src/Screens/Main/Chat/Title.tsx

```diff
@@ -101,15 +128,20 @@ const styles = RN.StyleSheet.create({
   },
   userIcon: {
     tintColor: colors.black,
-    width: 32,
-    height: 32,
+    width: 38,
+    height: 38,
```

Comment:
In this project, we want to use multiples of 8. So please change to 40 if possible.